docs/sources/datasources/mysql/_index.md

```diff
@@ -181,6 +181,8 @@ The response from MySQL can be formatted as either a table or as a time series.
 In the dataset dropdown, choose the MySQL database to query. The dropdown is be populated with the databases that the user has access to.
 When the dataset is selected, the table dropdown is populated with the tables that are available.
 
+NOTE: If a default database has been chosen through the Data Source configuration page, the Dataset Dropdown will be disabled, and only that configured database will be available for querying.
```

Comment:
Same with using bold here. and also `only the configured database` would make more sense

Reply:
Updated.